src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -65,7 +65,7 @@ <h2>Active Courses</h2>
               {{ courseStats[course.id].unregistered }}
             </span>
           </td>
-          <td class="text-center">
+          <td class="text-center bin-header-button">
```

Comment:
This one should also be `actions-cell` for the same reason as above.

Then, in `instructor-courses-page.component.scss`, you should add:
```
.actions-cell button {
    margin: 2px;
}
```